NoSleep/README.md

```diff
@@ -0,0 +1,62 @@
+<p align="center"><a href="https://github.com/3raxton/NoSleep/"
+target="_blank"><br><img width="100" src="https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/240/apple/232/sleeping-face_1f634.png"></a></p>
+<h1 align="center">No Sleep</h1>
+<p align="center">A fresher dark theme with a splash of red for Spotify thanks to <a href="https://github.com/khanhas/spicetify-cli" target="_blank"> spicetify-cli</a></p>
+<p align="center">
+<a href="https://twitter.com/3raxton"><img src="https://img.shields.io/badge/Contact-@3raxton-blue.svg" alt="License">
+</a>
+<a><img src="https://img.shields.io/badge/theme-dark-brightgreen.svg" alt="Theme"></a>
+<a><img src="https://img.shields.io/badge/no-sleep-red.svg" alt="No Sleep"></a>
+<a href="https://3raxton.github.io/license"><img src="https://img.shields.io/badge/License-MIT-blue.svg" alt="License">
+</a>
+
+</p>
+
+## Summary
+- 😴 A fresher dark theme with a splash of red
+- 👀 Easy on the eyes
+- 🎉 Open source
+- 🍁 If you like the No Sleep theme please consider <a href="https://www.patreon.com/3raxton" target="_blank"> supporting me on Patreon</a>
+
+![No Sleep](https://github.com/3raxton/NoSleep/blob/master/NoSleep/NS1.png?raw=true)
+![No Sleep](https://github.com/3raxton/NoSleep/blob/master/NoSleep/NS2.png?raw=true)
+
+## How to make No Sleep your Spotify theme
+
+1. [Install spicetify-cli using the instructions found here](https://github.com/khanhas/spicetify-cli/wiki/Installation)
+2.  Click on the green `Clone or download` button at the <b>top right</b> of the repo the No Sleep theme is in  and choose ```Download ZIP```
+3. Open the spicetify-cli themes folder (paths can be found below)
+  ```
+MacOS: ~/spicetify_data/Themes
+Windows: %userprofile%\.spicetify\Themes\
+Linux: $XDG_CONFIG_HOME/.config/spicetify/Themes/ or ~/.config/spicetify/Themes
+  ```
+1. Drag the ```NoSleep``` folder into the spicetify-cli Themes folder
+2. Find your ```config.ini``` file within the spicetify folder
+3. Set the theme to ```NoSleep``` in the ```config.ini``` folder
+4. Set desired theme in your `config.ini`:
+```ini
+[Setting]
+...
+current_theme = NoSleep
+...
+```
+1. Apply your theme to Spotify by typing the following in Terminal:<br> ```spicetify update``` <br>```spicetify apply``` 
+2. spicetify will apply the theme and restart Spotify 
+3. Try out the new theme and see if you like it
+4. You're done 🎉
```

Comment:
I think this is not necessary since it is already specified in the main README how to install a switch among the themes

Reply:
I agree, my fault since I was copying it over from the README in my repo. Just removed it